core-web/libs/block-editor/src/lib/extensions/ai-content-actions/ai-content-actions.component.html

```diff
@@ -0,0 +1,16 @@
+<p-listbox
+    [options]="actionOptions"
+    [style]="{
+        width: '12.5rem',
+        padding: '0.5rem',
+        'margin-left': '3.75rem',
+        'background-color': '#fff'
+    }"
```

Comment:
Can we switch to CSS `classes`? That way we can use our CSS variables

Reply:
Couldnt find another way to apply those styles on this listbox unless this way...

Reply:
You can pass the property `styleClass` to the `<p-listbox` and then style the class.

Reply:
See PrimeNg API for [Listbox](https://primeng.org/listbox#api.listbox) Component, we have [styleClass](https://primeng.org/listbox#api.listbox.props.styleClass) and [listStyleClass](https://primeng.org/listbox#api.listbox.props.listStyleClass) Properties.